src/control/server/ctl_system.go

```diff
@@ -198,7 +202,8 @@ func (svc *ControlService) SystemQuery(ctx context.Context, pbReq *ctlpb.SystemQ
 		Method: control.PingRanks,
 		Hosts:  pbReq.GetHosts(),
 		Ranks:  pbReq.GetRanks(),
-	}, true)
+		Force:  false, // ping harness only, not over drpc
```

Comment:
"Force" doesn't seem right for this. Maybe "Forward" to indicate it is forwarded to the data plane?

Reply:
this is a little bit of a hack, I'm using Force because it's already there as part of RanksReq, which is a generic request used by the different system command requests

Reply:
What does Force normally do? If it's used differently elsewhere, I'd lean against overloading it.

Reply:
Force implements SIGKILL Rather than TERM when RanksReq is passed through SystemStop, the request parameter isn't used when handling SystemPing so I'm using it to imply an invasive rather than non-invasive ping (where non-invasive uses the harness instance state and invasive pings over dRPC), I don't think we need to add another parameter that only applies to SystemPing, I think having handler specific usage of the force flag to reduce bloating is okay.

Reply:
That's a fair argument. Since "force" sounds a bit odd for choosing how far to send a ping, I'd still personally prefer the param be renamed (something like "aggressive" might be appropriate for both cases), but I can live with it as-is.

Reply:
appreciated, this area might be subject to change fairly soon during 
https://jira.hpdd.intel.com/browse/DAOS-6058 so I will keep those thoughts in mind